classification/metrics/compute_vessel_metrics.py

```diff
@@ -1073,15 +988,37 @@ def dump_html(args, results):
 this_dir = os.path.dirname(os.path.abspath(__file__))
 temp_dir = os.path.join(this_dir, 'temp')
 
+# TODO: compute the fraction of each vessel type, then compute
+# the weights for precision recall, accuracy with weighted
+# values rather than equal values.
+# Query for weights looks like:
+'''
+with
+
+core as (
+select * from `machine_learning_dev_ttl_120d.vessel_char_vid_features_v20190509*`
```

Comment:
@bitsofbits you shouldn't be querying a table in a ttl dataset, if this needs to be re-run move it to a permanent location.

Reply:
This should be a passed in parameter anyway. Gone now.